src/runtime.md

```diff
@@ -60,14 +60,14 @@ let component_id = builder.component(
 );
 ```
 
-%% ### Hot reload
+### Hot reload
 
-%% To disable hot reloading set the documents `hot_reload = false`.
+To disable hot reloading set the documents `hot_reload = false`.
 
-%% ```rust,ignore
-%% let mut doc = Document::new("@index");
-%% doc.hot_reload = false;
-%% ```
+```rust,ignore
+let mut doc = Document::new("@index");
+doc.hot_reload = false;
+```
```

Comment:
I don't think it's possible to disable hot reloading right now so perhaps we should leave this one commented out.

Reply:
Unfortunately the %% weren't acting as comments and it was just formatting weirdly: 
![image](https://github.com/user-attachments/assets/31f5dba6-a53d-45b5-b71e-1c749ed99fbc)